### DISCUSSION TITLE
Playing a video only once

### DISCUSSION THREAD

**Discussion by headquarter8302** [rocksdanister/lively#2376] [Q&A] [answered]
Hello all.

I am trying to make an MP4 video file play exactly only once, and after it has reached its last frame, pause itself and become static. I am unable to find any resources or settings that enables me to do this. Is there anything I'm missing?

**Comment by rocksdanister** [comment, marked as answer]
You can write a script:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls

Sine this is a niche feature there will not be a direct setting for this.

**Reply by headquarter8302** [reply]
That's a shame. Alas, thank you for showing me how to do it
